pkgs/os-specific/linux/kernel/patches.nix

```
@@ -19,6 +19,16 @@
       patch = ./bridge-stp-helper.patch;
     };
 
+  # Reverts the buggy commit causing https://bugzilla.kernel.org/show_bug.cgi?id=217802
+  dell_xps_regression = {
+    name = "dell_xps_regression";
+    patch = fetchpatch {
```

For next time, don't worry about this now unless you want to:

```suggestion
    patch = fetchurl {
```

You're fetching a static file, so there's no need for normalisation and stuff.  (And it's best to avoid that where possible as what "normalised" means depends on a specific version of patchutils.)

Alternatively, if there aren't any conflicts, you could `fetchpatch` the original commit and pass `revert = true;`, which is nice and declarative.